src/lib/locale/locales.js

```diff
@@ -178,7 +180,7 @@ export function getLocale (key) {
         key = [key];
     }
 
-    return chooseLocale(key);
+    return chooseLocale(key.map(normalizeLocale));
```

Comment:
IE8 doesn't support `.map`

Reply:
Thanks. BTW Where can I get the `browserslist` of the `moment` project? How could I know what is the browser version matrix that a contributor is supposed to support?

Reply:
Replaced `.map` by for-loop.